src/pages/App.svelte

```diff
@@ -26,7 +26,6 @@
   let showInstructions = false;
 
   const params = new URLSearchParams(window.location.search);
-  // TODO Add validation and some kind of error page
```

Comment:
I personally would leave this... we could later work on a bit of paranoia for other URL query params like `style` and `schema`. They probably warrant less intrusive handling, like just quietly switching to a default